web/src/services/theme/reearthTheme/common/metrics.ts

```diff
@@ -1,6 +1,6 @@
 const metrics = {
-  propertyMenuMinWidth: 272,
-  propertyMenuMaxWidth: 336,
+  propertyMenuWidth: 350,
```

Comment:
from design, this should be 235(left) or 286(right)

Reply:
@lavalse  would you please help us here in the new beta design  i noticed that the initial size for panels is 308 and we set the min size to 200 is that ok

Reply:
approved by red @lavalse